.doc_gen/metadata/cognito-identity_metadata.yaml

```diff
@@ -34,12 +43,21 @@ cognito-identity_CreateIdentityPool:
             - description: Create a new identity pool.
               snippet_tags:
                 - cognitoidentity.swift.create-identity-pool
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/cognito
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - cognito.java2.create_identity_pool.main
   services:
     cognito-identity: {CreateIdentityPool, ListIdentityPools}
 cognito-identity_DeleteIdentityPool:
-  title: Delete an &COGID; with a specified name
+  title: Delete an &COG; identity pool
```

Comment:
If we went with `&COGID;`, we'd have  

> Delete an Amazon Cognito Identity identity pool

And even if we went with

> Delete an &COGID; pool

we'd have 

> Delete an Amazon Cognito Identity pool

This whole section it turns out was mislabeled, since it's all about identity pools, not identities. I think the phrase should be **Amazon Cognito identity pools** like the first sentence on the [Using identity pools page](https://docs.aws.amazon.com/cognito/latest/developerguide/identity-pools.html).

Reply:
&COGID; is the entity for the service, but if you look everywhere in the documentation, when 'identity pools' are discussed, it's always 'Amazon Cognito indentity pools', paralleling 'Amazon Cognito user pools'.